ambari-server/src/main/resources/Ambari-DDL-MySQL-CREATE.sql

```diff
@@ -26,16 +26,6 @@ delimiter ;
 
 # USE @schema;
 
--- Set default_storage_engine to InnoDB
--- storage_engine variable should be used for versions prior to MySQL 5.6
-set @version_short = substring_index(@@version, '.', 2);
-set @major = cast(substring_index(@version_short, '.', 1) as SIGNED);
-set @minor = cast(substring_index(@version_short, '.', -1) as SIGNED);
-set @engine_stmt = IF(@major >= 5 AND @minor>=6, 'SET default_storage_engine=INNODB', 'SET storage_engine=INNODB');
-prepare statement from @engine_stmt;
-execute statement;
-DEALLOCATE PREPARE statement;
-
```

Comment:
Will this change our default behavior on Mysql 5.6 if we delete these?

Reply:
There will be an prerequisite that admin has to set mysql default engine to INNODB.

Reply:
Will it be better if we can simply replace the condition with `(@major >= 5 AND @minor>=6) or @major >= 8`?
Since mysql8 is also use `default_storage_engine` as variable name but the current condition cannot cover it?
I'm not sure if we will meet unexpected behaviors in old versions if we delete these.

Reply:
From mysql 5.7 onwards the default engine is INNODB and storage_engine variable is not available for mysql8

Reply:
I know. I mean, the condition `@major >= 5 AND @minor>=6` we are using now returns false when mysql version is 8.0, we can simple use `(@major >= 5 AND @minor>=6) or @major >= 8` to make it returns true, and avoid behavior changes when mysql version lower than 5.6

Reply:
ok. but I think no one will use it as it has many vulnerabilities and we can update wiki for supported version like we have for the java.

Reply:
We can remove support when the compatibility work of older versions is heavy.